release.md

```diff
@@ -0,0 +1,298 @@
+**DO NOT MERGE** - this is here so people can do review stuff to it if they want to.
+
+⚠️ **Early draft. Details subject to change.** ⚠️
+
+# 🐘 🐘 V5 🐘 🐘  🐘
+
+## 1. All SOLC.
```

Comment:
Let's remove the indexes for this headers, and instead have a table of contents at the top. There's a lot here. It'd be nice to lay it all out upfront.

Reply:
Ah very good idea!!

Reply:
Put a small TOC in but it doesn't scroll - not sure if it's because it's in a comment or the links are bad. Probably need to work on this in the release context.